netbox/data_source_netbox_ip_addresses_test.go

```diff
@@ -22,12 +22,14 @@ resource "netbox_ip_address" "test" {
   interface_id = netbox_interface.test.id
   status = "active"
   tags = [netbox_tag.test.name]
+	role = "anycast"
```

Comment:
this looks wrongly indented, did you mix spaces and tabs or is this a github?

Reply:
Seems fixed by running `gofmt -w`